app/build.gradle

```diff
@@ -26,7 +26,7 @@ checkstyle {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestCompile('com.android.support.test.espresso:espresso-intents:2.2.2', {
```

Comment:
I think this should be *added* to the build file and not replace espresso-core. Could be required by a test someone else makes.